### PR TITLE
Preparation for release v2021.2 (GUI)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

* Updates Changes file
* Bumps version packages.json and version.ts
* Updates installation doc with updated version

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+v3.4.0 2021-12-03 (public release version)
+
+ [Features]
+ - Replaces CentOS with Rocky Linux (#280)
+ - Provides translation to error messages from RPC API (#269, #239)
+ - Adds support from information banner (#266)
+ - Adds filter for test history (#253, #251)
+ - Adds run time configuration (#267, #260, #219, #170)
+ - Adds flexible URL (#252, #250)
+ - Updates Danish translation (#248, #244)
```
Comment (mattias-p) [inline Changes:13]: How come Danish translation updates are listed under Features but updates to other languages are listed under Fixes?

File: Changes [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+ - Adds support for translated tooltips (#243, #241)
+
+ [Fixes]
+ - Updates Finnish translation (#290, #288)
+ - Updates Swedish translation (#286)
+ - Updates version of Node.js used (#274)
+ - Fixes translation issue in browsers (#278)
+ - Fixes colors on buttons (#270)
+ - Improves on norrow device (#259)
```
Comment (mattias-p) [inline Changes:22]: `norrow` -> `narrow`

File: Changes [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+ - Adds run time configuration (#267, #260, #219, #170)
+ - Adds flexible URL (#252, #250)
+ - Updates Danish translation (#248, #244)
+ - Adds support for translated tooltips (#243, #241)
+
+ [Fixes]
+ - Updates Finnish translation (#290, #288)
+ - Updates Swedish translation (#286)
+ - Updates version of Node.js used (#274)
+ - Fixes translation issue in browsers (#278)
```
Comment (mattias-p) [inline Changes:20]: Doesn't `in browsers` go without saying?

**Review by mattias-p — APPROVED** [review]

File: Changes (hunk 1 above)
Comment (matsduf) [reply Changes:13]: Mistake. Fixed.

File: Changes (hunk 2 above)
Comment (matsduf) [reply Changes:22]: Fixed.

File: Changes (hunk 3 above)
Comment (matsduf) [reply Changes:20]: I do not understand. I removed that item and moved `#278` to "Improves translation handling".

**Review by mattias-p — APPROVED** [review]

**Pull request merged**